kupogo.go

```diff
@@ -163,54 +163,67 @@ func (c *Client) GetMatches(pattern string) (*Matches, error) {
 	return matches, nil
 }
 
-func (c *Client) GetMetadata(slotNo int, txId string) (*Metadata, error) {
+func (c *Client) GetMetadata(slotNo int, txId string) ([]*Metadata, error) {
```

Comment:
If the only use of Metadata returns a list, we should make Metadata a list.

```
type MetadataItem struct {
    Hash...
}

type Metadata []MetadataItem
```
Then, we just return *Metadata, error here